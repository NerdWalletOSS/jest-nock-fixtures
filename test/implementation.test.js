const fetch = require('node-fetch');
// import fetch from 'node-fetch';

const { expect } = global;

// this server responds with a current-time-in-ms in each response body
const TEST_URL = 'http://worldclockapi.com/api/json/utc/now';

describe('implementation', () => {
  it('should not allow external requests in CI', async () => {
    const res = await fetch(TEST_URL);
    const json = await res.json();
    expect(json).toMatchSnapshot();
  });
});
