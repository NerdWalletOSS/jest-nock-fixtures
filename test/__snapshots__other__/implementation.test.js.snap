// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`implementation AND YET here is another 1`] = `
Object {
  "$id": "1",
  "currentDateTime": "2021-03-31T15:21-04:00",
  "currentFileTime": 132616776883643760,
  "dayOfTheWeek": "Wednesday",
  "isDayLightSavingsTime": true,
  "ordinalDate": "2021-90",
  "serviceResponse": null,
  "timeZoneName": "Eastern Standard Time",
  "utcOffset": "-04:00:00",
}
`;

exports[`implementation AND YET here is another 2`] = `
Object {
  "$id": "1",
  "currentDateTime": "2021-03-31T15:21-04:00",
  "currentFileTime": 132616776886143890,
  "dayOfTheWeek": "Wednesday",
  "isDayLightSavingsTime": true,
  "ordinalDate": "2021-90",
  "serviceResponse": null,
  "timeZoneName": "Eastern Standard Time",
  "utcOffset": "-04:00:00",
}
`;

exports[`implementation AND here is another 1`] = `
Object {
  "completed": false,
  "id": 1,
  "title": "delectus aut autem",
  "userId": 1,
}
`;

exports[`implementation Hi here is another 1`] = `
Object {
  "completed": false,
  "id": 1,
  "title": "delectus aut autem",
  "userId": 1,
}
`;

exports[`implementation should not allow external requests in CI 1`] = `
Object {
  "$id": "1",
  "currentDateTime": "2021-03-31T19:21Z",
  "currentFileTime": 132616920876299360,
  "dayOfTheWeek": "Wednesday",
  "isDayLightSavingsTime": false,
  "ordinalDate": "2021-90",
  "serviceResponse": null,
  "timeZoneName": "UTC",
  "utcOffset": "00:00:00",
}
`;
